website_oca_integrator/views/website_oca_integrator_templates.xml

```diff
@@ -6,9 +6,9 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 <odoo>
 
     <!-- Add integrators link in footer -->
-    <template id="integrator_footer" inherit_id="website.footer_default"
+    <template id="integrator_footer" inherit_id="website.footer_custom"
```

Comment:
Is this shown by default?

Reply:
Default template doesn't exist and this the only footer on same module so I guess is shown by default

Reply:
Please make sure about this.

Reply:
Yes,in this commit footer_default is removed
https://github.com/odoo/odoo/commit/b2fefbd3c56b4ac1ef4cfe0387db45d24aa5a037

Reply:
Thanks @sergio-teruel